<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pedalCal</class>
 <widget class="QDialog" name="pedalCal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**************LABEL**************/
QLabel
{
	font: 16pt &quot;Futura PT&quot;;
	color: rgb(242, 242, 242);
	background-color: transparent;
}

/*----------- QPushButton ------------*/
QPushButton
{
	background: rgb(60, 60, 60);
	color: rgb(245, 245, 245);
	border: 2px solid rgb(245, 245, 245);
	font: 12pt &quot;Futura PT&quot;;
	padding: 0, 1, 0, 0;
}


QPushButton:pressed
{
	background: rgb(245, 245, 245);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(245, 245, 245);
	font: 12pt &quot;Futura PT&quot;;
	padding: 0, 1, 0, 0;
}

/*----------- QComboBox ------------*/
QComboBox
{
	font: 12pt &quot;Futura PT&quot;;
	border: 2px solid rgb(245, 245, 245);
	background: rgb(60, 60, 60);
	color: rgb(242, 242, 242);
	padding: 1px 3px 1px 4px;
 }

QComboBox QListView
{
	background-color: rgb(60, 60, 60);
	border: transparent;
	font: 12pt &quot;Futura PT&quot;;
	padding-left: 3px;
	padding-right: 3px;
	min-width: 70px;
}

QComboBox QAbstractItemView
{
    min-width: 150px;
}

QComboBox::item:alternate
{
	background: rgb(90, 90, 90);
}

QComboBox::item:selected
{
	border: 1px solid transparent;
	background: rgb(0,174,239);
}

QComboBox::indicator
{
	background-color: transparent;
	selection-background-color: transparent;
	color: transparent;
	selection-color: white;
}

QComboBox::drop-down
{
	width: 1px;
	height: 1px;
	subcontrol-origin: margin;
	subcontrol-position: right center;
	border-left-width: 0px;
	border-right-width: 0px;
	border-top-width: 0px;
	border-bottom-width: 0px;
	padding: 1px 1px 1px 4px;
}

QComboBox::disabled
{
	font: 12pt &quot;Futura PT&quot;;
	border: 2px solid rgb(100, 100, 100);
	background: rgba(20, 20, 20, 100);
	color: rgb(100, 100, 100);
	padding: 1px 3px 1px 4px;
}

QComboBox::focus
{
	border: 2px solid rgb(0, 174, 239);
}
</string>
  </property>
  <widget class="QSlider" name="slider_min">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>191</y>
     <width>60</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-------QSlider-------*/
QSlider
{
	background: none;
}

/*-----VERTICAL-----*/

QSlider::groove:vertical
{
	border: none;
}

QSlider::handle:vertical
{
	background: rgba(0, 0, 255,255);
	border: none;
	height: 9px;
	
}

QSlider::add-page:vertical
{
	background: rgba(89, 89, 89,255);
}

QSlider::sub-page:vertical
{
	background: rgba(169, 169, 169, 0);
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_max">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>36</y>
     <width>60</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-------QSlider-------*/
QSlider
{
	background: none;
}

/*-----VERTICAL-----*/

QSlider::groove:vertical
{
	border: none;
}

QSlider::handle:vertical
{
	background: rgba(255, 0, 0,255);
	border: none;
	height: 9px;
	
}

QSlider::add-page:vertical
{
	background: rgba(89, 89, 89,0);
}

QSlider::sub-page:vertical
{
	background: rgba(89, 89, 89,255);
}</string>
   </property>
   <property name="minimum">
    <number>155</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="bg_midde_slider">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>36</y>
     <width>60</width>
     <height>255</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(169, 169, 169, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="slider_in">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>36</y>
     <width>60</width>
     <height>255</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-------QSlider-------*/
QSlider
{
	background: none;
}

/*-----VERTICAL-----*/

QSlider::groove:vertical
{
	border: none;
}

QSlider::handle:vertical
{
	background: rgba(255, 255, 255, 255);;
	border: none;
	height: 9px;
	
}

QSlider::add-page:vertical
{
	background: rgba(89, 89, 89,255);
}

QSlider::sub-page:vertical
{
	background: rgba(89, 89, 89,255);
}</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="ReadOnlySlider" name="slider_out">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>36</y>
     <width>60</width>
     <height>255</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-------QSlider-------*/
QSlider
{
	background: none;
}

/*-----VERTICAL-----*/

QSlider::groove:vertical
{
	border: none;
}

QSlider::handle:vertical
{
	background: rgba(0, 255, 0,255);;
	border: none;
	height: 9px;
	
}

QSlider::add-page:vertical
{
	background: rgba(89, 89, 89,255);
}

QSlider::sub-page:vertical
{
	background: rgba(89, 89, 89,255);
}</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_min_val">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>291</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid blue;</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_in_val">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>291</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_out_val">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>291</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_max_val">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>11</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid red;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_in">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>11</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid white;</string>
   </property>
   <property name="text">
    <string>IN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_out">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>11</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string>OUT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_instructions">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>35</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: rgba(0,0,0,255);
    color: white;
    font: 14px &quot;Futura-normal&quot;;
border: black;
}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Futura-normal'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;1. Plug in your expression pedal.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;2. Adjust the expression pedal from heel to toe and observe the live values on the left slider. The middle slider adjusts the min and max thresholds, and the right slider shows the calibrated output value.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;3. Put the expression pedal at the zero (heel) position. Adjust the min threshold (middle slider, bottom/blue handle) until the output slider value is zero.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;4. Put the expression pedal at the max (toe) position. Adjust the max threshold (middle slider, top/red handle) until the output slider value is 127.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;5. When satisfied, click [save] to send the calibration values to the device.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Droid Sans Mono'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>10</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: white;
	font: 18pt &quot;Futura PT&quot;;
	color: rgb(242, 242, 242);
	background-color: transparent;
}

</string>
   </property>
   <property name="text">
    <string>Expression Pedal Calibration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="dropdown_table">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the type of curve to &lt;br&gt;apply to incoming values. </string>
   </property>
   <property name="styleSheet">
    <string notr="true">/*----------- QComboBox ------------*/
QComboBox
{

}

QComboBox::down-arrow
{
	border: none;
	background: none;
	image: url(:/ui/ui_images/combobox_menu_arrows.svg);
	padding-right: 15px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cosine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exponential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Logarithmic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeadZone</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Curve:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_restore">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>332</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>332</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>332</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="no_focus">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>no_focus</zorder>
  <zorder>bg_midde_slider</zorder>
  <zorder>slider_min</zorder>
  <zorder>slider_max</zorder>
  <zorder>slider_in</zorder>
  <zorder>slider_out</zorder>
  <zorder>label_min_val</zorder>
  <zorder>label_in_val</zorder>
  <zorder>label_out_val</zorder>
  <zorder>label_max_val</zorder>
  <zorder>label_in</zorder>
  <zorder>label_out</zorder>
  <zorder>text_instructions</zorder>
  <zorder>label_title</zorder>
  <zorder>dropdown_table</zorder>
  <zorder>label_13</zorder>
  <zorder>button_restore</zorder>
  <zorder>button_cancel</zorder>
  <zorder>button_save</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReadOnlySlider</class>
   <extends>QSlider</extends>
   <header>readOnlySlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
